estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,24 @@
+from odoo import api, models, fields
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+
+    price = fields.Float()
+    status = fields.Selection(copy=False, selection=[('accepted', 'Accepted'), ('refused', 'Refused')])
+    partner_id = fields.Many2one('res.partner', required=True)
+    property_id = fields.Many2one('estate.property', required=True)
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(compute='_compute_date_deadline', inverse="_inverse_date_deadline")
+
+    @api.depends("validity", "create_date")
+    def _compute_date_deadline(self):
+        for record in self:
+            if not record.create_date:
+                record.create_date = fields.Date.today()
+            record.date_deadline = fields.Date.add(record.create_date, days=record.validity)
+
+    def _inverse_date_deadline(self):
+        for record in self:
+            record.validity = (record.date_deadline - record.create_date.date()).days
```

Comment:
```suggestion
            date = offer.create_date.date() if offer.create_date else fields.Date.today()
            offer.validity = (offer.date_deadline - date).days
```
create_date might not be set yet